packages/core/database/lib/entity-manager/index.js

```diff
@@ -429,13 +465,14 @@ const createEntityManager = (db) => {
         }
 
         if (attribute.joinColumn && attribute.owner) {
+          const relIdsToAdd = toIds(cleanRelationData.set);
           if (
             attribute.relation === 'oneToOne' &&
             isBidirectional(attribute) &&
-            data[attributeName]
+            relIdsToAdd.length
```

Comment:
This is a question maybe for @alexandrebodin . why we only add the relation when its a `oneToOne` when using `joinColumn`

Reply:
We actually add it for oneToOne or oneToMany (look a bit below in the code :)) 

This usecase is just to unset the relations on oneToOne because it must be maintain as a unique on both side contrary to oneToMany where it will be an additive operation